internal/proxy/hook/authz/authz.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/goto/shield/core/relation"
 	"github.com/goto/shield/core/resource"
 	"github.com/goto/shield/core/user"
+	proxyattr "github.com/goto/shield/internal/proxy/attribute"
```

Comment:
Why should we use`proxyattr`?

Reply:
in this file, attribute package from opentelemetry is imported so we use `proxyattr` alias.